AidanStuff/File Converter/File Converter/FileOptions.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+using Microsoft.VisualBasic.FileIO;
+namespace File_Converter
+{
+    public class FileOptions
+    {
+        
+        //Change file ending/convert file
+        public static void ChangeFileProperties(int prop)
```

Comment:
The 'prop' parameter appears to have to possible values, where each value has a specific meaning. Instead of declaring its type as 'int', this is a good candidate for defining an enum type. E.g.,

enum PropertyId
{  
    Extension,  
    Name      
}  
  
public static void ChangeFileProperties(PropertyId prop)  
{  
    . . .  
    switch (prop)  
    {  
        case PropertyId.Extension:  
            . . . etc.